frontend/src/libs/components/checkbox/checkbox.tsx

```diff
@@ -0,0 +1,38 @@
+import { type ReactElement } from 'react';
+import {
+    type Control,
+    type FieldPath,
+    type FieldValues,
+    useController,
+} from 'react-hook-form';
+
+type CheckboxProperties<T extends FieldValues> = {
+    name: FieldPath<T>;
+    label: string;
+    control: Control<T>;
+};
+
+const Checkbox = <T extends FieldValues>({
+    name,
+    label,
+    control,
+}: CheckboxProperties<T>): ReactElement => {
+    const { field } = useController<T>({ name, control });
```

Comment:
You can use useFormController that's already in the hooks folder. Path: bundles -> common -> hooks